starport/templates/app/stargate/config.yml.plush

```diff
@@ -14,3 +14,16 @@ client:
 faucet:
   name: bob
   coins: ["5token", "100000stake"]
+genesis:
```

Comment:
Can we remove this section for now, this adds too much to the existent config.yml format. We can discuss to add it back or even make it not required at all.

Reply:
I removed the empty parameters, but I think we can keep the chain-id because it is required to publish the chain. WDYT?

Reply:
I couldn't understand sorry. Chain id can be also set by the flag in the publish command which means it'll cancel this one here.